app/models/topic_timer.rb

```diff
@@ -119,73 +161,63 @@ def cancel_auto_silent_close_job
   end
 
   def cancel_auto_publish_to_category_job
-    Jobs.cancel_scheduled_job(:publish_topic_to_category, topic_timer_id: id)
+    Jobs.cancel_scheduled_job(TopicTimer.type_job_map[:publish_to_category], topic_timer_id: id)
```

Comment:
I think we need to change the use of `TopicTimer.type_job_map` across the other methods as well :)

Reply:
I would prefer to keep it as is until we can deprecate completely the old scheduled jobs, this method here is why I have made the map:

```
https://github.com/discourse/discourse/pull/11698/files#diff-875f29b2d419e2d435ada228303b71455e3b8106a495d0ddce30bfe7b24fb68aR67-R79
```

So in a few months we can remove this method and the map entirely.

Reply:
ok :+1: